### PR TITLE
docs: add key and desc field in hertz doc head

### PR DESCRIPTION
**Pull request by chaoranz758**
#### What type of PR is this?

docs

#### Check the PR title.

- [x] This PR title match the format: \<type\>(optional scope): \<description\>
- [x] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional):

### CODE REVIEW

**Comment by GuangmingLuo** [comment]
@li-jin-gou @rogerogers Please help to review this  PR

File: content/zh/docs/hertz/getting-started/_index.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 title: "快速开始"
 linkTitle: "快速开始"
 weight: 2
-description: >
+keywords: ["Golang", "Go", "Hertz", "开发环境", "快速上手", "代码生成工具"]
```
Comment (li-jin-gou) [inline content/zh/docs/hertz/getting-started/_index.md:5]: "Golang", "Go" 这个关键词没用吧

File: content/zh/docs/hertz/overview/_index.md [hunk 2]
```diff
@@ -2,7 +2,8 @@
 title: "概览"
 linkTitle: "概览"
 weight: 1
-description: >
+keywords: ["HTTP", "Golang", "Go", "Hertz", "架构设计", "框架特点", "框架性能", "相关项目", "相关文章"]
```
Comment (li-jin-gou) [inline content/zh/docs/hertz/overview/_index.md:5]: "Golang", "Go" "相关项目", "相关文章"  +1

File: content/zh/docs/hertz/tutorials/basic-feature/adaptor.md [hunk 3]
```diff
@@ -2,13 +2,14 @@
 title: "适配器"
 date: 2023-01-11
 weight: 16
-description: >
+keywords: ["适配器", "http.Request", "http.ResponseWriter", "net/http"]
```
Comment (li-jin-gou) [inline content/zh/docs/hertz/tutorials/basic-feature/adaptor.md:5]: 这个很准确

File: content/zh/docs/hertz/tutorials/basic-feature/middleware/_index.md [hunk 4]
```diff
@@ -2,6 +2,7 @@
 title: "中间件概览"
 date: 2022-05-20
 weight: 6
+keywords: ["中间件", "服务端中间件", "客户端中间件", "路由级别"]
 description: >
```
Comment (li-jin-gou) [inline content/zh/docs/hertz/tutorials/basic-feature/middleware/_index.md:6]: 没有 description

File: content/zh/docs/hertz/tutorials/basic-feature/middleware/gzip.md [hunk 5]
```diff
@@ -3,7 +3,8 @@
 title: "Gzip 压缩"
 date: 2022-09-01
 weight: 4
-description: >
+keywords: ["Gzip 压缩", "压缩"]
```
Comment (li-jin-gou) [inline content/zh/docs/hertz/tutorials/basic-feature/middleware/gzip.md:6]: "Gzip", "压缩"

**Comment by li-jin-gou** [comment]
英文和中文都修改一下

File: content/en/docs/hertz/tutorials/basic-feature/middleware/basic-auth.md [hunk 6]
```diff
@@ -2,7 +2,8 @@
 title: "Basic Auth"
 date: 2022-10-13
 weight: 2
-description: >
+keywords: ["HTTP", "Basic Auth"]
+description: "Hertz provides an [implementation](https://github.com/cloudwego/hertz/tree/main/pkg/app/middlewares/server/basic_auth) of Basic Auth, referencing gin's [implementation](https://github.com/gin-gonic/gin#using-basicauth-middleware)."
```
Comment (rogerogers) [inline content/en/docs/hertz/tutorials/basic-feature/middleware/basic-auth.md:6]: 同上

**Review by rogerogers — COMMENTED** [review]
desc里有很多链接，会导致desc长度太长。

**Comment by li-jin-gou** [comment]
看了一下问题都改了， @GuangmingLuo @rogerogers 再看下没什么问题先合进去吧

**Review by rogerogers — APPROVED** [review]

**Review by li-jin-gou — APPROVED** [review]

**Pull request merged**